component/vshn_postgres.jsonnet

```diff
@@ -17,6 +17,9 @@ local defaultUser = 'postgres';
 local defaultPort = '5432';
 
 
+local serviceNameLabelKey = 'appcat.vshn.io/servicename';
+local serviceNamespaceLabelKey = 'appcat.vshn.io/claim-namespace';
```

Comment:
I'm a bit confused with the name of this label. Why do we use `claim-namespace` here? Would it make sense to add `service` word in the label?

Reply:
I don't have strong feelings about that name. We just need to somehow have a label that tells us in which namespace this corresponding claim is for billing. `claim-namespace` seemed like an obvious choice, but we can use whatever we want

Reply:
I thought thoroughly about this label and realised the name is actually what we want. Maybe I could add a word that the purpose of this label is for the billing. Now, not sure if you want to include that in the group or in the name of the label.